src/python/grpcio/grpc/experimental/aio/_channel.py

```diff
@@ -20,12 +21,18 @@
 
 class UnaryUnaryMultiCallable(aio.UnaryUnaryMultiCallable):
 
-    def __init__(self, channel, method, request_serializer,
-                 response_deserializer):
+    def __init__(self, channel: cygrpc.AioChannel, method, request_serializer,
+                 response_deserializer) -> None:
         self._channel = channel
         self._method = method
         self._request_serializer = request_serializer
         self._response_deserializer = response_deserializer
+        self._loop = asyncio.get_event_loop()
+
+    def _timeout_for_deadline(self, timeout=None):
```

Comment:
Is there a reason why timeout is a keyword argument when it's basically the only argument?